src/main/java/com/google/sps/data/DatastoreEntityToObjectConverter.java

```diff
@@ -49,8 +51,40 @@ protected static Post convertPost(Entity postEntity)
         long answerId = (long) (postEntity.getProperty(Constants.POST_ANSWER));
         Comment answer = convertComment(answerId);
         List<Comment> comments = convertCommentsFromPost(postEntity); 
+        String tab = (String) postEntity.getProperty(Constants.POST_TAB);
         long id = postEntity.getKey().getId();
-        return new Post(question, answer, comments, id); 
+        return new Post(question, answer, comments, tab, id); 
+    }
+
+    /**
+     * Converts a tab entity into a Tab object 
+     * @param tabEntity entity of the tab 
+     * @throws EntityNotFoundException
+     * @return the Tab object 
+     */ 
+    protected static Tab convertTab(Entity tabEntity) 
```

Comment:
can these be package private instead since there is no subclass?

Reply:
I get a maven error if I make it private

Reply:
not private, package private (aka not writing a modifier)

https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html.

I'm pretty sure it can be package private.